packages/@react-spectrum/s2/src/Field.tsx

```diff
@@ -202,7 +202,8 @@ export const FieldGroup = forwardRef(function FieldGroup(props: FieldGroupProps,
         }
       }}
       onTouchEnd={e => {
-        if (!(e.target as Element).closest('button,input,textarea')) {
+        let target = e.target as HTMLElement;
```

Comment:
test for this change?

Reply:
I added a test for textarea, but couldn't get the datepicker test to quite simulate the proper touch end event on the content editable for some reason...